log/sequencer_test.go

```diff
@@ -201,15 +201,15 @@ func newSignerWithErr(signErr error) (gocrypto.Signer, error) {
 }
 
 func createTestContext(ctrl *gomock.Controller, params testParameters) (testContext, context.Context) {
-	mockStorage := storage.NewMockLogStorage(ctrl)
+	mockStorage := &stestonly.FakeLogStorage{}
```

Comment:
So we can't use mocks for this any more? But we're still calling them mocks. Might be better to rename the variables though I know it's fiddly.

Reply:
Yeah :/
I left the names deliberately to reduce diff count, but I've updated them now - PTAL?